www/src/pages/insights.jsx

```diff
@@ -181,18 +205,29 @@ export default function InsightsPage() {
           <h1>Usage Insights</h1>
           <p>Last updated: {new Date(analysisLastUpdated).toLocaleDateString()}</p>
         </header>
-        <Tabs defaultActiveKey="summary" id="uncontrolled-tab-example">
-          <Tab eventKey="summary" title="Summary">
+        <Tabs
+          activeKey={tabs.includes(path) ? path : tabs[0]}
```

Comment:
I do think we'll need to figure out a solution to the tabs routing here so that it's consistently functioning. If I'm on `/insights?tab=components` and refresh the page, for example, the URL still says I'm on "Components" tab though I'm actually now on the "Summary" tab. Then, when trying to navigate to the "Components" tab again, I'm unable to.